src/elements/content-sidebar/activity-feed/annotations/__tests__/AnnotationActivityMenu.test.js

```diff
@@ -30,12 +36,16 @@ describe('elements/content-sidebar/ActivityFeed/annotations/AnnotationActivityMe
         expect(wrapper.find('DeleteConfirmation').length).toEqual(1);
     });
 
-    test('shoud render resin tags', () => {
-        const wrapper = getWrapper({ canDelete: true });
+    test.each`
+        canDelete | canEdit  | dataTestId                      | dataResinTarget
+        ${true}   | ${false} | ${'delete-annotation-activity'} | ${'activityfeed-annotation-delete'}
```

Comment:
Are the permutations of permissions meaningful for this test? Can they both be hardcoded to true?

Reply:
Yes they can both be true - it's possible that we have both menu items show up during the render.

Reply:
This test isn't checking their effect on each other, just that their individual resin tags are correct. We can pass both permissions and true and check both resin tags in one test.